docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -300,7 +306,7 @@ function template_preprocess_card(&$variables) {
     $variables['headline'] = [
       'headline_text' => $variables['title'],
       'headline_level' => $variables['title_heading_size'] ?: 'h2',
-      'headline_class' => 'headline',
+      'headline_class' => $title_headline_classes ?: 'headline',
```

Comment:
We can just add the `headline` class before we get to this point so we don't need the null coalescing operator.